src/meshdb/settings.py

```diff
@@ -52,6 +52,51 @@
 
 USE_X_FORWARDED_HOST = True
 
+SECURE_HSTS_SECONDS = 30  # TODO: Increase me to 31536000 https://github.com/nycmeshnet/meshdb/issues/642
+SECURE_HSTS_PRELOAD = True
+SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+
+SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+SECURE_REFERRER_POLICY = "strict-origin-when-cross-origin"
+
+CSP_REPORT_ONLY = True  # TODO: Set me to false https://github.com/nycmeshnet/meshdb/issues/644
+CSP_DEFAULT_SRC = [
+    "'self'",
+    "*.nycmesh.net",
+    "maps.googleapis.com",
+    "maps.gstatic.com",
+    "fonts.googleapis.com",
+    "fonts.gstatic.com",
+    "'unsafe-inline'",  # TODO: Remove me https://github.com/nycmeshnet/meshdb/issues/645
+    "*.browser-intake-datadoghq.com",
+]
+CSP_REPORT_URI = [
+    "https://csp-report.browser-intake-datadoghq.com/api/v2/logs"
+    "?dd-api-key=pubca00a94e49167539d2e291bea2b0f20f&dd-evp-origin=content-security-policy"
```

Comment:
Is this naughty? We give it out for free to anyone who hits the site so having it in code seems fine?

Reply:
Verify correct type and limited access

Reply:
This is definitely a client token, from [the docs](https://docs.datadoghq.com/account_management/api-app-keys/#client-tokens) we are definitely using it correctly:

> For security reasons, API keys cannot be used to send data from a browser, mobile, or TV app, as they would be exposed client-side. Instead, end user facing applications use client tokens to send data to Datadog.

I can't find any knobs to turn for the permissions on the token, and nothing is mentioned in the docs about this. I think client tokens are inherently limited to only specific things (probably to prevent footguns)